test/Player-test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
```

Comment:
It's convention to require outside libraries first and then have your own requires (like `Player.js`) right underneath, separated by a blank line.

```js
const chai = require('chai');
const expect = chai.expect;

const Player = require('../js/player.js');
const Question = require('../js/Question.js');
const data = require('../js/data.js');
```
Be sure to also rename your files for your classes so that the filename is capitalized (e.g. `player.js` should be `Player.js`)